site/_scss/layouts/devsite-theme.scss

```diff
@@ -1,9 +1,877 @@
+      --devsite-landing-row-group-grid-column: 1 / span 3;
+      --devsite-landing-row-header-grid-column: 1 / span 3;
+      --devsite-landing-row-header-max-width: 100%;
+    }
+
+    @media (max-width: 591px) {
+      --devsite-h1-font: 700 1.5625rem #{$TEXT_PRIMARY_FONT_FAMILY};
+
+      .dcc-blog-card {
+        max-width: 100%;
+        margin-bottom: 2rem;
+      }
+    }
+  }
+
+  
+
+  .dcc-docs {
+    --devsite-landing-group-padding: 4.6875rem 1.5rem 0;
+    --devsite-landing-group-margin: 1.5rem auto;
+    --devsite-landing-row-header-padding: 0.5rem 0;
+    --devsite-buttons-margin: 0.5rem 0 0 0;
+    --devsite-item-gap: 2.5rem;
+
+    position: relative;
+
+    &::before {
+      background: linear-gradient(
+        92deg,
+        #fb0301 0%,
+        #fb0301 10.94%,
+        #fbbb06 19.79%,
+        #fbbb06 37.5%,
+        #34a653 60.42%,
+        #34a653 76.56%,
+        #206ace 100%
+      );
+      content: "";
+      height: 3px;
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: 100%;
+    }
+
+    @media (max-width: 591px) {
+      --devsite-h1-font: 500 1.75rem #{$TEXT_PRIMARY_FONT_FAMILY};
+    }
+  }
+
+  [background="pale-blue"] {
+    --devsite-background-color: #{$BG_PRIMARY};
+    --devsite-button-border: none;
+    --devsite-button-border-hover: none;
+    --devsite-card-background: white;
+    --devsite-heading-color: #{$TEXT_PRIMARY_PALEBLUE};
+    --devsite-h1-font: 500 3rem #{$TEXT_PRIMARY_FONT_FAMILY};
+    --devsite-landing-row-group-margin: 0;
+    --devsite-landing-row-header-text-margin: 0 auto 3.75rem auto;
+    --devsite-landing-row-header-text-max-width: 57.8125rem;
+    --devsite-landing-row-padding: 3.5rem 0;
+
+    margin: 3rem 0 2.1875rem;
+
+    h2{
+      text-align: center;
+    }
+
+    @media (max-width: 1280px) {
+      --devsite-h1-font: 500 2.25rem #{$TEXT_PRIMARY_FONT_FAMILY};
+
+      header + div {
+        grid-template-columns: 1fr 1fr;
+      }
+    }
+
+    @media (max-width: 591px) {
+      --devsite-h1-font: 500 1.75rem #{$TEXT_PRIMARY_FONT_FAMILY};
+
+      header + div {
+        grid-template-columns: 1fr;
+      }
+    }
+  }
+
+  .dcc-social {
+      --devsite-item-gap: 3.125rem;
+      --devsite-landing-row-header-text-margin: 0 0 2.875rem 0;
+      --devsite-landing-row-padding: 3.438rem 0 6.25rem;
+      --devsite-h1-font: 700 3rem #{$TEXT_PRIMARY_FONT_FAMILY};
+
+      margin: 0;
+
+      h2 {
+        text-align: left;
+      }
+
+      h3 + div {
+        color: #606367;
+      }
+
+      .button {
+        border-radius: 0;
+      }
+
+      img {
+        --devsite-item-image-width: 3.563rem;
+      }
+
+      @media (max-width: 1280px) {
+        --devsite-h1-font: 700 2.5rem #{$TEXT_PRIMARY_FONT_FAMILY};
+      }
+
+      @media (max-width: 591px) {
+        --devsite-h1-font: 700 1.75rem #{$TEXT_PRIMARY_FONT_FAMILY};
+      }
+    }
+
+  footer {
+    margin-top: 0;
+  }
+}
+
+/* Layout, header, and footer */
+[theme=web-theme] {
```

Comment:
Since these styles do not influence the look and feel of the content from devsite, please remove to another file.

Reply:
We moved all code in `/* Layout, header, and footer */ ` out, and replaced the current layout with the devsite layout following Matthias' feedback so the whole site now has the new header and the new side nav for mobile.